detection-rules/attachment_qr_link_b64_recipient.yml

```
@@ -0,0 +1,42 @@
+name: "Attachment: QR Code Link With Base64-Encoded Recipient Address"
+description: "Detects when an image or macro attachment contains QR codes that, when scanned, lead to URLs containing the recipient's email address. This tactic is used to uniquely track or target specific recipients and serve tailored credential phishing pages."
+type: "rule"
+severity: "high"
+source: |
+  type.inbound
+  and 1 of (
+    any(ml.nlu_classifier(subject.subject).intents,
+        .name == "cred_theft" and .confidence == "high"
+    ),
+    body.current_thread.text is null,
+    any($org_slds, strings.icontains(sender.display_name, .))
+  )
+  and any(attachments,
+          (.file_type in $file_types_images or .file_extension in $file_extensions_macros or .file_type == "pdf")
+          and any(file.explode(.),
+                  any(recipients.to,
+                      any(beta.scan_base64(..scan.qr.url.url, ignore_padding=true),
```

add check to ensure the recipient email domain is valid.
```suggestion
                      .email.domain.valid
                      and any(beta.scan_base64(..scan.qr.url.url, ignore_padding=true),
```